apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagService.kt

```diff
@@ -287,6 +305,21 @@ class RealGrunnlagService(
     )
 }
 
+private fun HistorikkForeldreansvar.tilGrunnlagsopplysning(
+    fnr: Folkeregisteridentifikator
+): Grunnlagsopplysning<JsonNode> {
+    return Grunnlagsopplysning(
+        id = UUID.randomUUID(),
+        kilde = Grunnlagsopplysning.Pdl("pdl", Tidspunkt.now(), null, null),
```

Comment:
Foreslår å setje iallfall navn="pdl" som standardverdi for parameteren her. Kjens litt dobbelt opp at vi sender inn "pdl" som streng kvar gong vi bruker denne.

Reply:
Ja, god ide!